src/Version.js

```diff
@@ -0,0 +1 @@
+export const version = "v1.0.1-pre.0";
```

Comment:
You can import the generated Spago files to avoid hardcoding this, e.g. [like it's done here](https://github.com/purescript/spago/blob/a4fc6b3cfd3410e8cd0ecb2cf547bda6568aac1c/src/Spago/BuildInfo.purs#L77)

Reply:
The file is sitting in `.spago/BuildInfo.purs`

Reply:
Wonderful!